webapp_provider/dockerfile

```diff
@@ -12,4 +12,4 @@ RUN npm install
 COPY . .
 
 EXPOSE 5000
-CMD ["npx", "serve","-s","public"]
+CMD echo $REST_PORT > /usr/src/app/public/backend.cfg && npx serve -s public
```

Comment:
i think it could be just public `echo $REST_PORT >public/backend.cfg && npx serve -s public`
because thats what workspace does. but i mean there is nothing wrong with doing it that way.